.golangci.yaml

```diff
@@ -10,7 +10,6 @@ linters:
   enable:
     - asciicheck
     - bodyclose
-    - deadcode
```

Comment:
should the other linters be added to the commented section below?

Reply:
I move the ones that were relevant to the generic issue. The ones that have been deleted are: 

`[WARN] x is deprecated. Replaced by unused.`

I can include them commented out if we want to ever incorporate them again if they are picked up by a maintainer

Reply:
Ah no need if they're deprecated 👍